src/browser/mod.rs

```diff
@@ -493,6 +493,33 @@ pub fn default_executable() -> Result<std::path::PathBuf, String> {
     Err("Could not auto detect a chrome executable".to_string())
 }
 
+// Returns the latest chrome revision for the current platform.
+// This function will panic on unsupported platforms.
+#[cfg(feature = "fetch")]
+pub fn latest_chrome_revision() -> Result<String> {
+    let mut url = format!("https://storage.googleapis.com/chromium-browser-snapshots");
+
+    if cfg!(target_os = "linux") {
+        url = format!("{}/Linux_x64/LAST_CHANGE", url);
+    } else if cfg!(all(target_os = "macos", target_arch = "aarch64")) {
+        url = format!("{}/Mac_Arm/LAST_CHANGE", url);
+    } else if cfg!(all(target_os = "macos", not(target_arch = "aarch64"))) {
+        url = format!("{}/Mac/LAST_CHANGE", url);
+    } else if cfg!(windows) {
+        url = format!("{}/Win_x64/LAST_CHANGE", url);
+    } else {
+        // This code will panic on unsupported platforms.
+        #[allow(unreachable_code)]
+        {
+            unimplemented!("This platform is not supported")
+        }
+    }
```

Comment:
maybe it would be better to refactor this into a match statement ?

Reply:
Refactoring to match statement looks good. And reflecting the definition suggested by @masc-it looks good.

Reply:
I'm sorry I closed this PR by mistake. (;_;)
please check https://github.com/rust-headless-chrome/rust-headless-chrome/pull/380